calebwagner/python-server
# POST HTTP method addition

#### Changes Made
1. Modified file `animals/request.py`, `locations/request.py`, `employees/request.py`, and `customers/request.py` to include `POST method` functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-ch4
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > run python server
    > check for updated create functions within request.py modules
4. View code file.
    > Confirm file modifications are present as indicated above.
